actionpack/lib/action_dispatch/railtie.rb

```diff
@@ -39,7 +44,7 @@ class Railtie < Rails::Railtie # :nodoc:
       ActionDispatch::ExceptionWrapper.rescue_responses.merge!(config.action_dispatch.rescue_responses)
       ActionDispatch::ExceptionWrapper.rescue_templates.merge!(config.action_dispatch.rescue_templates)
 
-      config.action_dispatch.authenticated_encrypted_cookie_salt = "authenticated encrypted cookie" if config.action_dispatch.use_authenticated_cookie_encryption
+      config.action_dispatch.authenticated_encrypted_cookie_salt ||= "authenticated encrypted cookie"
```

Comment:
Let's move this to the alongside the other `_salt`s up top and skip the `||=` for `=`.